playbook-website/app/javascript/site_styles/docs/_kit_doc.scss

```diff
@@ -153,3 +155,47 @@
 pre.highlight {
   padding: $space_xl;
 }
+
+/* VERTICAL SECTION SEPERATOR */
+
+.fixedHeightSSeparator {
+  height: 200px;
+  width: 30px;
+  float: left;
+}
+
+.SFixedHeight {
+  float: left;
+  width: 200px;
+}
+
+.clear {
+  clear: both;
+}
+
+.autoHeightSSeperator {
+  height: 100vh;
+  width: 100px;
+  float: left;
+}
+
+.itemSSeperator {
+  float: left;
+  width: 50%;
+}
+
+.SSeperatorscroll {
+  width: 100%;
+  height: 50px;
+  overflow: scroll;
+}
+
+.flexSSeperatorContainer {
+  height: 100px;
+}
+
+.flexSSeperatortext {
+  width: 50%;
```

Comment:
The whole point of this story is to document different ways of getting the vertical separator to display WITHOUT adding extra sass. Please remove. CC @kre8sions

Reply:
@jasoncypret I made the changes to less scss, they just affect the text now and not the vertical separator.  

Also on a different note there are a lot of different files attached to this PR that deal with status colors, I am not sure what that is about, but should go to a different PR. Maybe?

Reply:
I don't know how  my PR ([PLAY-74] Allow Legend Kit to use our Status Colors (#1725)) ended up in your `08b7d6b` commit.